package.json

```diff
@@ -16,22 +16,30 @@
     "@babel/preset-env": "^7.5.5",
```

Comment:
Please delete the package-lock.json file! You should only have a yarn.lock. the package-lock.json file was created because someone ended up using an `npm` command

Reply:
Removed!